### PR TITLE
added more data and better instructions

### PR DESCRIPTION
**Pull request by gu0255ti-s**

### CODE REVIEW

File: java/src/FilterData.java [hunk 1]
```diff
@@ -16,22 +16,33 @@ private String getPath(String fileName) {
...
+        System.out.println("--------------------------");
         String pattern = sc.nextLine(); // reads string.
-
+        System.out.println("--------------------------");
         String fileName = getPath(name+".txt");
 
-        // read file into stream, try-with-resources
+        // read file into stream and filter based on filter pattern
         try (Stream<String> stream = Files.lines(Paths.get(fileName))) {
             stream.filter(l -> l.contains(pattern))
```
Comment (Schyllert) [inline java/src/FilterData.java:38]: test

File: java/src/FilterData.java [hunk 2]
```diff
@@ -16,16 +16,20 @@ private String getPath(String fileName) {
 
     private void run() {
 
+        //Input filename
         Scanner sc = new Scanner(System.in); // System.in is a standard input stream.
-        System.out.print("Enter a filename (without .txt): ");
+        System.out.println("Enter a filename (without .txt): ");
+        System.out.println("avicii or catData");
```
Comment (Schyllert) [inline java/src/FilterData.java]: Not clear what you mean with this: if you are giving a suggestion to file names be more clear.

Comment (gu0255ti-s) [reply java/src/FilterData.java]: This is intended to give the user suggestions for what is a possible entry at this stage.

Comment (gu0255ti-s) [reply java/src/FilterData.java]: Is it unclear for the user or for a developer?

Comment (Schyllert) [reply java/src/FilterData.java]: Maybe just add "possible file names to choose from: avicii or catData" and it will be clearer for users and developers

Comment (gu0255ti-s) [reply java/src/FilterData.java]: Ok!

Comment (gu0255ti-s) [reply java/src/FilterData.java]: I have commited a fix to this comment, please review

**Review by Schyllert — APPROVED** [review]
Looks good

**Pull request merged**